### PR TITLE
COMMERCE-6525 & COMMERCE-6524 Add sitemap providers for products and categories

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/sitemap/AssetCategorySitemapURLProvider.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.content.web.internal.sitemap;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/sitemap/AssetCategorySitemapURLProvider.java:15]: @alecsloan we can just have this in the util folder

```suggestion
package com.liferay.commerce.product.content.web.internal.util;
```

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/sitemap/CPDefinitionSitemapURLProvider.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.content.web.internal.sitemap;
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/sitemap/CPDefinitionSitemapURLProvider.java:15]: @alecsloan we can just have this in the util folder

```suggestion
package com.liferay.commerce.product.content.web.internal.util;
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Review by dsitu — APPROVED** [review]

**Comment by alecsloan** [comment]
Thanks @dsitu!

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
Sent to Brian here: https://github.com/brianchandotcom/liferay-portal/pull/102106

**Pull request closed without merging**